backends/dpdk/dpdkMetadata.cpp

```diff
@@ -101,15 +101,13 @@ const IR::Node *DirectionToRegRead::postorder(IR::DpdkListStatement *l) {
     return l;
 }
 
-const IR::Node *DirectionToRegRead::postorder(IR::DpdkAction *a) {
```

Comment:
This might still be required, right? 
Since actions are not inlined into the apply block, we might still need to do the conversion here. Please check with a testcase We can get it validated by the backend team.

Reply:
no, because it already will be initialized in apply block, action can use without any issue